src/Microsoft.Health.Fhir.Core/Configs/ExportJobConfiguration.cs

```diff
@@ -52,5 +52,10 @@ public class ExportJobConfiguration
         /// </summary>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "This is a configuration class")]
         public IList<ExportJobFormatConfiguration> Formats { get; set; }
+
+        /// <summary>
+        /// Determines max size for an exported file.
+        /// </summary>
+        public uint MaxFileSizeInMB { get; set; } = 64;
```

Comment:
That whole name and description reads for me:
`We not gonna export more than this limit`
while your intent is more a rolling file size.

So I would suggest to rename it RollingMaxFileSize.

Also I somewhat feel it needs to be part of the `ExportJobFormatConfiguration` since you define your file name pattern via format

Reply:
I have updated the name to `RollingFileSizeInMB`. But I have kept the config location in `Export` since I have updated the code to use this outside the export format. The sequence will now be added to every file irrespective of the Export format.